src/command/supergraph/config.rs

```diff
@@ -52,6 +51,12 @@ mod tests {
     use std::fs;
 
     #[test]
+    fn it_can_fail_a_test() {
+        panic!("it failed")
+    }
```

Comment:
i... don't think that this will make CI fail?

Reply:
I cannot find this test in the current tests that are running either. Are we ignoring this file/set of tests anywhere?

Reply:
this is the complete output i get from `cargo test` without running through xtask. you can see the test is run and says FAILED, but the reason it fails is not expanded, and cargo exits with code 0.

```console
$ cargo test
warning: unused import: `assert_fs::TempDir`
  --> src/command/supergraph/config.rs:48:9
   |
48 |     use assert_fs::TempDir;
   |         ^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

warning: unused import: `camino::Utf8PathBuf`
  --> src/command/supergraph/config.rs:49:9
   |
49 |     use camino::Utf8PathBuf;
   |         ^^^^^^^^^^^^^^^^^^^

warning: unused import: `std::convert::TryFrom`
  --> src/command/supergraph/config.rs:50:9
   |
50 |     use std::convert::TryFrom;
   |         ^^^^^^^^^^^^^^^^^^^^^

warning: unused import: `std::fs`
  --> src/command/supergraph/config.rs:51:9
   |
51 |     use std::fs;
   |         ^^^^^^^

warning: 4 warnings emitted

    Finished test [unoptimized + debuginfo] target(s) in 1m 53s
     Running unittests (target/debug/deps/rover-81890668a6f7b8c9)

running 39 tests
test command::docs::shortlinks::tests::can_get_url_from_slug ... ok
test command::graph::publish::tests::handle_response_doesnt_err ... ok
test command::subgraph::delete::tests::handle_response_doesnt_error_with_all_successes ... ok
test command::docs::shortlinks::tests::can_make_shortlink_vec_from_map ... ok
test command::subgraph::delete::tests::handle_response_doesnt_error_with_all_failures ... ok
test command::subgraph::publish::tests::handle_response_doesnt_error_with_all_failures ... ok
test command::subgraph::publish::tests::handle_response_doesnt_error_with_all_successes ... ok
test error::metadata::suggestion::test::possible_values_match ... ok
test error::metadata::suggestion::test::possible_values_nomatch ... ok
test command::supergraph::config::tests::it_can_fail_a_test ... FAILED       <----------------------- HERE
test utils::env::tests::it_can_set_and_read_from_mock ... ok
test utils::env::tests::it_can_remove_from_mock ... ok
test utils::env::tests::it_parses_config_home ... ok
test utils::git::tests::does_not_mind_case ... ok
test utils::git::tests::removed_user_from_remote_with_only_user ... ok
test command::config::auth::tests::it_can_set_custom_api_key ... ok
test utils::git::tests::returns_none_unrecognized_protocol ... ok
test utils::git::tests::strips_usernames_from_ssh_urls ... ok
test utils::git::tests::does_not_allow_remotes_from_unrecognized_providers ... ok
test utils::git::tests::works_with_special_chars ... ok
test utils::loaders::tests::load_schema_from_flag_errs_on_bad_path ... ok
test utils::loaders::tests::load_schema_from_flag_loads ... ok
test utils::loaders::tests::load_schema_from_stdin_works ... ok
test utils::parsers::tests::it_correctly_parses_path_option ... ok
test utils::parsers::tests::it_correctly_parses_stdin_flag ... ok
test utils::parsers::tests::it_errs_with_empty_path ... ok
test utils::git::tests::works_with_non_url_github_remotes ... ok
test utils::git::tests::works_with_not_url_bitbucket_remotes ... ok
test utils::git::tests::works_with_non_url_gitlab_remotes ... ok
rover-config-list 0.1.7
List all configuration profiles

USAGE:
    rover config list [OPTIONS]

FLAGS:
    -h, --help    Prints help information

OPTIONS:
    -l, --log <log-level>    Specify Rover's log level [possible values: error, warn, info,
                             debug, trace]
     Running unittests (target/debug/deps/rover-49ff35d4d46b66b4)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running tests/info.rs (target/debug/deps/info-3e02978dd4cef836)

running 1 test
test it_prints_info ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.90s

     Running tests/installers.rs (target/debug/deps/installers-21143de2e1bea13d)

running 2 tests
test it_has_windows_installer ... ok
test it_has_nix_installer ... ok

test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.13s

     Running tests/main.rs (target/debug/deps/main-a18bd87116969357)

running 6 tests
test schema::fetch::it_has_a_graph_fetch_command ... ok
test its_executable ... ok
test config::api_key::it_has_a_config_profile_auth_command ... ok
test config::api_key::it_errors_on_an_empty_apikey ... ok
test config::profile::it_can_list_one_profile ... ok
test config::profile::it_can_list_no_profiles ... ok

test result: ok. 6 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.39s

   Doc-tests rover

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

$ echo $?
0